Cubical/Categories/Constructions/Quotient/More.agda

```diff
@@ -37,25 +37,26 @@ module _ (C : Category ℓ ℓ') where
       QF = QuoFunctor C _~_ ~refl ~cong
 
     open Categoryᴰ
+    open Section
     module _ (Dᴰ : Categoryᴰ C/~ ℓD ℓD') where
       private
         module Dᴰ = Categoryᴰ Dᴰ
-
-      -- TODO: make a macro that applies to Eq.refl in exactly this way
-      reindexᴰQuo : Categoryᴰ C ℓD ℓD'
-      reindexᴰQuo = reindex' Dᴰ QF Eq.refl λ _ _ → Eq.refl
+      module ReindexQuo = EqReindex Dᴰ QF Eq.refl (λ _ _ → Eq.refl)
 
       open Section
-      elim : (F : Section reindexᴰQuo)
+
+      -- TODO: should elim be the name for the global section or the local
```

Comment:
based on comments elsewhere, the default naming should be for local, right?

Reply:
I thought so but the local `elim`s get you into transport hell...

Reply:
I think what I'll do is that `elim` will mean a global section (which matches how `elim` for e.g., sums works) and call the other one `elimLocal`